components/widgets/icons/close_icon.jsx

```diff
@@ -7,7 +7,10 @@ import {FormattedMessage} from 'react-intl';
 export default class CloseIcon extends React.PureComponent {
     render() {
         return (
-            <span {...this.props}>
+            <button
+                className='style--none close-x'
```

Comment:
Should we have these changes for the CloseIcon and modal css on a separate PR? This is not related to marketplace specifically and affects the modals in general.

Reply:
This change was causing issues updating the snapshots and blocking the CI build for this PR. I've moved these changes to this PR https://github.com/mattermost/mattermost-webapp/pull/3835